src/gui/plugins/global_illumination_civct/CiVctCascadePrivate.qml

```diff
@@ -0,0 +1,255 @@
+      cascadePtr.resolutionY = value;
+    }
+  }
+  IgnSpinBox {
+    property int oldValue: -1
+
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: resolutionSpinZ
+    value: cascadePtr.resolutionZ
+    enabled: GlobalIlluminationCiVct.cascadesEditable
+    minimumValue: 4
+    maximumValue: 512
+    decimals: 1
+    onEditingFinished: {
+      var tmpValue = value;
+      tmpValue = nearestPowerOf2(tmpValue, oldValue);
+      oldValue = tmpValue;
+      value = tmpValue;
+      cascadePtr.resolutionZ = value;
+    }
+  }
+
+  Text {
+    Layout.columnSpan: 6
+    id: octantCountStr
+    color: "dimgrey"
+    text: qsTr("OctantCount")
+  }
+
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: octantCountX
+    value: cascadePtr.octantCountX
+    minimumValue: 1
+    maximumValue: 8
+    decimals: 1
+    onEditingFinished: {
+      cascadePtr.octantCountX = value
+    }
+  }
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: octantCountY
+    value: cascadePtr.octantCountY
+    minimumValue: 1
+    maximumValue: 8
+    decimals: 1
+    onEditingFinished: {
+      cascadePtr.octantCountY = value
+    }
+  }
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: octantCountZ
+    value: cascadePtr.octantCountZ
+    minimumValue: 1
+    maximumValue: 8
+    decimals: 1
+    onEditingFinished: {
+      cascadePtr.octantCountZ = value
+    }
+  }
+
+  Text {
+    Layout.columnSpan: 4
+    id: thinWallCounterStr
+    color: "dimgrey"
+    text: qsTr("ThinWallCounter")
+  }
+
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: thinWallCounter
+    value: cascadePtr.thinWallCounter
+    minimumValue: 0
+    maximumValue: 5
+    decimals: 2
+    stepSize: 0.1
+    onValueChanged: {
+      cascadePtr.thinWallCounter = value
+    }
+  }
+
+  Text {
+    Layout.columnSpan: 6
+    color: "dimgrey"
+    text: qsTr("AreaHalfSize")
+  }
+
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: areaHalfSizeX
+    value: cascadePtr.areaHalfSizeX
+    enabled: GlobalIlluminationCiVct.cascadesEditable
+    minimumValue: 0.01
+    maximumValue: 100000
+    decimals: 2
+    onEditingFinished: {
+      cascadePtr.areaHalfSizeX = value
+    }
+  }
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: areaHalfSizeY
+    value: cascadePtr.areaHalfSizeY
+    enabled: GlobalIlluminationCiVct.cascadesEditable
+    minimumValue: 0.01
+    maximumValue: 100000
+    decimals: 2
+    onEditingFinished: {
+      cascadePtr.areaHalfSizeY = value
+    }
+  }
+  IgnSpinBox {
+    Layout.columnSpan: 2
+    Layout.fillWidth: true
+    id: areaHalfSizeZ
+    value: cascadePtr.areaHalfSizeZ
+    enabled: GlobalIlluminationCiVct.cascadesEditable
+    minimumValue: 0.01
+    maximumValue: 100000
+    decimals: 2
+    onEditingFinished: {
+      cascadePtr.areaHalfSizeZ = value
+    }
+  }
+}
+
+/*##^##
```

Comment:
remove?

Reply:
This is added by QtDesigner.

I can remove it but it keeps coming back. Should we leave it?